### PR TITLE
add cwl for package 'changes'

### PR DESCRIPTION
**Pull request by fkastner**
Hi,
I adapted the autogenerated cwl file for the 'changes' package to recognize author id's and the available keyvals.
I have left the long list that txs originally autogenerated at the end. Should that be removed instead?
Is anything else needed?

### CODE REVIEW

File: completion/changes.cwl [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+\setcommentmarkup{arg1}#S
+\setdeletedmarkup{arg1}#S
+\sethighlightmarkup{arg1}#S
+\setsocextension{arg1}#S
+\setsummarytowidth{arg1}#S
+\setsummarywidth{arg1}#S
+\settruncatewidth{arg1}#S
+\summaryofchangesname#S
+\theauthorcommentcount
+\\added#S
```
Comment (sunderme) [inline completion/changes.cwl:78]: I would prefer if the useless stuff below is removed (artifacts from the scan process)

File: completion/changes.cwl [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+\changesnoloc#S
+\changesnosoc#S
+\changesreplacename#S
+\cleaders#S
+\compactsummaryofchangesname#S
+\csname#S
+\ifIsEmpty{arg1}{arg2}{arg3}#S
+\listofchangesname#S
+\protect#S
+\setaddedmarkup{arg1}#S
```
Comment (sunderme) [inline completion/changes.cwl:65]: these commands( \set...)  are mentioned in the manual, at least #S should be removed

**Comment by fkastner** [comment]
I updated the file according to your review.

**Comment by sunderme** [comment]
looks good
thanks

**Pull request merged**